modules/affinity/src/main/java/org/apache/ignite/internal/affinity/AffinityManager.java

```diff
@@ -52,9 +72,121 @@ public AffinityManager(
         this.configurationMgr = configurationMgr;
         this.metaStorageMgr = metaStorageMgr;
         this.baselineMgr = baselineMgr;
+
+        String localNodeName = configurationMgr.configurationRegistry().getConfiguration(NetworkConfiguration.KEY)
+            .name().value();
+
+        configurationMgr.configurationRegistry().getConfiguration(LocalConfiguration.KEY)
```

Comment:
I believe that we should listen `ClusterConfiguration.metastorageClusterNodeNames ` instead of local ones.

Reply:
Replaced to cluster.